test/decorator.js

```diff
@@ -33,3 +33,15 @@ test('fastify.swagger should throw if called before ready (openapi)', async (t)
 
   t.throws(fastify.swagger.bind(fastify))
 })
+
+test('decorator can be overridden', async (t) => {
+  t.plan(1)
+  const fastify = Fastify()
+
+  await fastify.register(fastifySwagger, { decorator: 'customSwaggerDecorator' })
+
+  await fastify.ready()
```

Comment:
why two ready?

Reply:
Wanted to be really ready /s 😄 

I removed the extra one.

Reply:
😂